py/server/deephaven/table.py

```diff
@@ -362,9 +366,24 @@ def _j_py_script_session() -> _JPythonScriptSession:
 _numpy_type_codes = ["i", "l", "h", "f", "d", "b", "?", "U", "O"]
 
 
+def _array_component_type(fn) -> str:
+    sig = inspect.signature(fn)
+    t = sig.return_annotation
+    if isinstance(t, _GenericAlias) and issubclass(t.__origin__, Sequence):
+        return np.dtype(t.__args__[0]).char
```

Comment:
What if the user does not provide an `__args__[0]` value in the type hint?  Does it default to giving `Any`, does it cause a syntax error, or do we get some un-understandable error here?

Reply:
if no `__args__` (meaning just `List`, not `List[float]`) is provided, then `isinstance(t, _GenericAlias)` would fail first.